src/routes/blog/post/what-is-mcp/+page.markdoc

```diff
@@ -0,0 +1,210 @@
+date: 2025-03-11
+cover: /images/blog/what-is-mcp/cover.png
+timeToRead: 7
+author: ebenezer-don
+category: tutorial
+---
+
+If you've ever tried using an AI assistant for something practical, like pulling real data from your work files, checking a database, or sending a message, then you've probably hit a frustrating wall.
+
+These AI models are brilliant, but they're also **disconnected** from the tools we actually use.
+
+You ask:
+
+- *“Summarize my latest report.”* → AI doesn't know where your reports are.
+- *“Check today's database entries.”* → AI can't see your database.
+- *“Post an update on Slack.”* → AI can't touch Slack.
+
+For all their intelligence, AI models are basically **brains with no hands**. They can think, but they can't **interact** with the world outside of what's in their training data.
+
+This is where you start to need **MCP (Model Context Protocol)**.
+
+And no, that name is not very intuitive. You've probably heard it thrown around on social media and in tech circles, but it doesn't make much sense on its own.
+
+**What does "Model Context Protocol" mean?** Let's break it down in a way that actually makes sense.
+
+# So, what's MCP?
+
+Forget the fancy words for a second.
+
+**MCP** is just a way for AI assistants to interact with real-world tools like databases, file systems, APIs, and apps.
+
+Currently, AI assistants can reason, process information, and answer questions, but don't know where anything is, and can't actually do anything outside of its box.
+
+Now imagine you give it a phone with direct lines to your company's database, your Slack account, your email inbox, and your documents.
+
+That's **exactly** what MCP does.
+
+It's the system that allows an AI assistant to:
+
+- **Fetch** real data.
+- **Look up** information from live sources.
+- **Trigger** actions (like posting messages, running queries, or updating records).
+
+Instead of just *guessing* based on past knowledge, the AI can now **pull actual facts** from the places that matter.
+
+MCP is just the **standardized way** of making this happen. Instead of every AI needing custom integrations for every tool, MCP provides **one universal way** to hook everything together.
+
+MCP was developed by Anthropic and released as an open standard in November 2024, making it a relatively new but rapidly growing solution.
+
+# How does MCP actually work?
+
+Let's say you have an AI assistant running on your computer, and you want it to check your company's database for the latest user signups.
+
+Without MCP:
+
+1. The AI assistant doesn't have direct access to the database.
+2. You'd have to manually open the database, find the right table, and read the data yourself.
+
+With MCP:
+
+1. The AI assistant sends a **request** to an MCP server:
+    > “Get the latest 10 users from the database.”
```

Comment:
@ebenezerdon is this intentional behaviour?
![image](https://github.com/user-attachments/assets/9ed10e95-f901-4abe-8117-74627714a8a8)

Reply:
Fixed